dataactcore/models/jobTrackerInterface.py

```diff
@@ -6,18 +6,12 @@
 from dataactcore.utils.statusCode import StatusCode
 from dataactcore.utils.responseException import ResponseException
 from dataactcore.utils.cloudLogger import CloudLogger
-from dataactcore.utils.jobQueue import JobQueue
-from dataactcore.config import CONFIG_JOB_QUEUE
+from dataactcore.utils.jobQueue import enqueue
 from dataactvalidator.validation_handlers.validationError import ValidationError
 
 
 class JobTrackerInterface(BaseInterface):
     """Manages all interaction with the job tracker database."""
-
-    def __init__(self):
-        self.jobQueue = JobQueue(job_queue_url=CONFIG_JOB_QUEUE['url'])
-        super(JobTrackerInterface, self).__init__()
```

Comment:
This constructor was setting up `self.jobQueue` and then calling it's parent class' constructor. By removing the `__init__` override, we'll continue to call the parent class' constructor (due to class inheritance) -- we're just not doing the "special" thing of setting `self.jobQueue`. Does that make sense?